client/cli/src/main.rs

```diff
@@ -70,14 +82,24 @@ fn main() {
         .clone()
         .unwrap_or_else(|| "".to_string());
     let mnemonic_file = args.mnemonic_file.clone();
+
+    // If waypoint is given explicitly, use its value, otherwise try to retrieve it from a given
+    // URL.
+    let waypoint = args.waypoint.or_else(|| {
```

Comment:
so this can still be None right? probably worth a comment

Reply:
If I specify both, should we check for the latest?